hmpps-prisoner-search/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/resource/AttributeSearchResource.kt

```diff
@@ -22,7 +22,15 @@ import uk.gov.justice.digital.hmpps.prisonersearch.search.resource.advice.ErrorR
 import uk.gov.justice.digital.hmpps.prisonersearch.search.services.attributesearch.AttributeSearchService
 import uk.gov.justice.digital.hmpps.prisonersearch.search.services.attributesearch.api.AttributeSearchRequest
 
-typealias AttributeTypes = Map<String, String>
+@Schema(description = "An attribute that can be searched for in a query")
+data class Attribute(
+  @Schema(description = "The name of the attribute to be used when searching", example = "firstName")
+  val name: String,
+  @Schema(description = "The type of the attribute (used to determine which matcher to use)", example = "String")
+  val type: String,
+  @Schema(description = "Whether the attribute search will be fuzzy. Generally applicable to String attributes containing free text such as names.", example = "true")
+  val fuzzySearch: Boolean,
+)
```

Comment:
The get attributes endpoints now tells you if a field supports fuzzy searches.

Reply:
would be good to explain what _fuzzy_ means - maybe at a higher level?

Reply:
just spotted you've done that below 😂